network-area-diagram-viewer/README.md

```diff
@@ -0,0 +1,10 @@
+# Area-diagram-viewer-rte
+
+Most updated version of the area diagram viewer
```

Comment:
I think this readme file can stay there, but you should also add a subsection in the root README.md of powsybl-dev-tools.